lib/file_codec_unittest.py

```diff
@@ -39,6 +39,10 @@ def EncodeCommandLine(self, parameters, bitrate, videofile, outputfile):
   def DecodeCommandLine(self, videofile, inputfile, outputfile):
     return 'cp %s %s' % (inputfile, outputfile)
 
+  def EncoderVersion(self):
+    return 'CopyingCodec'
```

Comment:
Maybe return 'CopyingCodec v1' just to make it a bit clearer it's a kind of version of the codec.